presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1974,6 +1970,12 @@ private static void renameDirectory(HdfsContext context, HdfsEnvironment hdfsEnv
         }
     }
 
+    private Table getTableOrElseThrow(HiveIdentity identity, String databaseName, String tableName)
```

Comment:
just call it `getTable`

Reply:
the method would then clash with other `getTable` that does return `Optional<Table>`

Reply:
`doGetTable` or something more creative.